packages/sources/svelte/src/components/overlays/VtmnToast/README.md

```diff
@@ -0,0 +1,43 @@
+# Quick start
+
+You need to add `@vtmn/svelte` on your `package.json`
+
+```
+npm install @vtmn/svelte
+```
+
+Now, you can use `VtmnToast` on your app.
+
+# How it works
+
+First, you need to place the `VtmnToast` component preferably at your
```

Comment:
First, you need to place the `VtmnToast` component on the top of your application